src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/utils/TransactionsViewProjectionHandler.kt

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.ecommerce.eventdispatcher.utils
+
+import it.pagopa.ecommerce.commons.documents.v2.Transaction
+import it.pagopa.ecommerce.eventdispatcher.repositories.TransactionsViewRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+
+@Component
+object TransactionsViewProjectionHandler {
+
+  lateinit var env: Environment
+
+  // Flag to decide if we have to update the transactions-view collection (true) or not.
+  const val ENV_TRANSACTIONSVIEW_UPDATE_ENABLED_FLAG = "transactionsview.update.enabled"
+
+  @Autowired
+  fun init(environment: Environment) {
+    env = environment
+  }
+
+  fun saveEventIntoView(
```

Comment:
```suggestion
  fun updateTransactionView(
```

🚲  method clarity: there is no event involved into this action

Reply:
renamed (inside a larger PR refactoring the code) in https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/426/commits/185368519f8c0a5760cb9cba733ed82f3ac48301